docs/remote_inference_blueprints/bedrock_connector_titan_embedding_blueprint.md

```diff
@@ -89,6 +89,72 @@ POST /_plugins/_ml/connectors/_create
 }
 ```
 
+If you are using AWS OpenSearch Service version 2.11, there are no built-in functions for pre_process_function and post_process_function.
+So, you need to add the script as shown below.
+
+```json
+POST /_plugins/_ml/connectors/_create
+{
+  "name": "Amazon Bedrock Connector: embedding",
+  "description": "The connector to bedrock Titan embedding model",
+  "version": 1,
+  "protocol": "aws_sigv4",
+  "parameters": {
+    "region": "<PLEASE ADD YOUR AWS REGION HERE>",
+    "service_name": "bedrock",
+    "model": "amazon.titan-embed-text-v1"
+  },
+  "credential": {
+    "roleArn": "<PLEASE ADD YOUR AWS ROLE ARN HERE>"
+  },
+  "actions": [
+    {
+      "action_type": "predict",
+      "method": "POST",
+      "url": "https://bedrock-runtime.us-east-1.amazonaws.com/model/amazon.titan-embed-text-v1/invoke",
+      "headers": {
+        "content-type": "application/json",
+        "x-amz-content-sha256": "required"
+      },
+      "request_body": "{ \"inputText\": \"${parameters.inputText}\" }",
+      "pre_process_function": """
```

Comment:
This format is not so friendly for someone who is using curl to create connector. How about change to one line string with escaping ?

Reply:
is goot idean. i will change